news/2024/2024-04-19-project-prism-2024-registrations-now-open.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Project Prism 2024: Registrations Now Open!"
+date: 2024-04-19 15:00:00 +0000
+---
+
+Welcome to Project Prism, a 1v1 open-ranked osu!taiko tournament with a unique Raid Battle mechanic!
+
+![](/wiki/shared/news/2024-04-19-project-prism-2024-registrations-now-open/banner.jpg)
+
+As the successor to the [osu!taiko Malaysian Tournament (OTMT)](/wiki/Tournaments/OTMT/2022) and [Global Taiko Malaysian Showdown (GTMS)](https://gtsosu.com/gtms/home) series, [Project Prism](https://prism.osumalaysia.org/) bursts through the gates endeavouring to deliever a brand-new osu! tournament experience, featuring a full storyline, SEVEN (7) original characters, music releases, artworks, and much more!
```

Comment:
```suggestion
As the successor to the [osu!taiko Malaysian Tournament (OTMT)](/wiki/Tournaments/OTMT/2022) and [Global Taiko Malaysian Showdown (GTMS)](https://gtsosu.com/gtms/home) series, [Project Prism](https://prism.osumalaysia.org/) bursts through the gates endeavouring to deliever a brand-new osu! tournament experience, featuring a full storyline, **seven** (7) original characters, music releases, artworks, and much more!
```
Perhaps bold this instead of capitalising. Could confuse people since other tournament abbreviations are used in the same sentence, but this one is not a tournament series.